packaging/test_irods_client_rest_cpp.py

```diff
@@ -68,6 +68,60 @@ def test_access_with_default_arguments(self):
             finally:
                 os.remove(file_name)
                 admin.assert_icommand(['irm', '-f', file_name])
+    def test_logical_path_rename(self):
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+        with session.make_session_for_existing_admin() as admin:
+            # test on a data object
+            pwd, _ = lib.execute_command(['ipwd'])
+            path = pwd + '/data_object'
+            admin.assert_icommand(['itouch', path])
+            new_path = path.replace('data_object', 'new_data_object')
+            res = irods_rest.logical_path_rename(
+                token,
+                path,
+                new_path
+            )
+            self.assertEqual(res, "")
+            self.assert_icommand(['irm', new_path])
+            # test on a collection
+            path = pwd + '/collection'
+            new_path = path.replace("collection", "new_collection")
+            admin.assert_icommand(['imkdir', path])
+            # make sure that it works with a collection that's actually populated
+            admin.assert_icommand(['itouch', path + '/data_object'])
+            res = irods_rest.logical_path_rename(
+                _token,
```

Comment:
Codacy says `_token` isn't defined, and I think I agree. Should this be `token` instead of `_token`? Same for 109 and 119.